Triphouse-project/.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode
```

Comment:
Good job,
but you can create folder `styles` where placed `scss` and `css` folder.
example of convertion: sass --watch styles/scss/main.scss styles/css/style.css
 styles/css/style.css add to index.html

Reply:
Replaced scss and css files in folder styles.